gensim/models/fasttext.py

```diff
@@ -243,6 +243,14 @@ def word_vec(self, word, use_norm=False):
     def load_fasttext_format(cls, *args, **kwargs):
         return Ft_Wrapper.load_fasttext_format(*args, **kwargs)
 
+    @classmethod
+    def load_word2vec_format(cls, fname, fvocab=None, binary=False, encoding='utf8', unicode_errors='strict',
```

Comment:
Necessary? Personally, I would leave this out. I don't see any problem with getting a "deprecation error" instead of "not implemented error". Plus, it's weird to call a method that you know will fail. This is likely to cause more confusion than good. @menshikh-iv, what do you think?

Reply:
Makes sense. Removing this function from here.

Reply:
@janpom agree with you